cms/djangoapps/contentstore/views/tests/test_organizations.py

```diff
@@ -0,0 +1,32 @@
+"""Tests covering the Organizations listing on the Studio home."""
+import json
+from mock import patch
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from util.organizations_helpers import add_organization
+from student.tests.factories import UserFactory
+
+
+@patch.dict('django.conf.settings.FEATURES', {'ORGANIZATIONS_APP': True})
+class TestOrganizationListing(TestCase):
+    """Verify Organization listing behavior."""
+    @patch.dict('django.conf.settings.FEATURES', {'ORGANIZATIONS_APP': True})
```

Comment:
I have to patch `ORGANIZATIONS_APP` both at class level and on `setup` since class level patch would not have any effect on `setup`. It works for all the tests in class but does not work for `setup` so had to patch it there.